.pre-commit-config.yaml

```diff
@@ -1,21 +1,18 @@
 -   repo: git://github.com/pre-commit/pre-commit-hooks
-    sha: 5bf6c09
+    sha: v1.1.1
     hooks:
     -   id: trailing-whitespace
         exclude: cherrypy/test/static/index.html
     -   id: flake8
-        args:
```

Comment:
I prefer multi line sequences, because they produce better diffs and are easily readable. (I also often synchronize contents via `vimdiff`)